manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: MSI.Kombustor.4
+PackageVersion: 4.1.27

```

```suggestion
PackageVersion: 4.1.27.0
```
Edit folder name as well

---

@Exorcism0666